Infrastructure/PluginExecutor.cs

```diff
@@ -112,9 +113,13 @@ private void Execute(string pluginId, string input, string user)
                 plugin.Execute(new PluginInput(input, persistentData, callbacks));
 
             }
-            catch (Exception)
+            catch (Exception ex)
             {
-                _messageSender.SendMessage(user, "An error occured while executing the plugin.");
+                _messageSender.SendMessage(user, "An error occured while executing the plugin, please type help again");
+                var plugin= new PluginInput(input, null, callbacks);
+                plugin.Callbacks.EndSession();
```

Comment:
Please format

Reply:
You can format the code, look here how
https://docs.microsoft.com/en-us/answers/questions/550115/automatic-format-code-in-visual-studio.html